src/snowcli/cli/stage.py

```diff
@@ -168,3 +168,28 @@ def stage_drop(
             name=name,
         )
         print_db_cursor(results)
+
+
+@app.command("remove")
+def stage_remove(
+    environment: str = EnvironmentOption,
+    stage_name: str = typer.Argument(..., help="Stage name"),
+    file_name: str = typer.Argument(..., help="File name"),
+):
+    """
+    Remove file from stage
+    """
+
+    env_conf = AppConfig().config.get(environment)
+
+    if config.is_auth():
+        config.connect_to_snowflake()
```

Comment:
Please refactor this to a the approach:
https://github.com/Snowflake-Labs/snowcli/blob/bb7d3d40427b1b19d0e0eec74416ce6bda2aa349/src/snowcli/cli/stage.py#L27-L32

Reply:
Updated @sfc-gh-turbaszek